docs_src/examples/app-service-examples/camera-management/Ch-CameraManagement.md

```diff
@@ -77,6 +77,13 @@ Install build tools:
 sudo apt install build-essential
 ```
 
+### Clone the EdgeX Examples repository
```

Comment:
I think this step should go down in step 2 to avoid confusion. Starting evam can be step 3.